schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_entry.json

```diff
@@ -10289,6 +10289,934 @@
+                  "rcsb_units": "angstroms",
+                  "rcsb_description": [
+                     {
+                        "text": "The data low resolution diffraction limit, in Angstroms, found in the input structure factor file.\nX-ray entry specific, obtained in the EDS step.",
+                        "context": "dictionary"
+                     }
+                  ]
+               },
+               "Fo_Fc_correlation": {
+                  "type": "number",
+                  "description": "Fo,Fc correlation: The difference between the observed structure factors (Fo) and the \ncalculated structure factors (Fc) measures the correlation between the model and the\nexperimental data. \nX-ray entry specific, obtained in the eds step from REFMAC calculation.",
+                  "rcsb_description": [
+                     {
+                        "text": "Fo,Fc correlation: The difference between the observed structure factors (Fo) and the \ncalculated structure factors (Fc) measures the correlation between the model and the\nexperimental data. \nX-ray entry specific, obtained in the eds step from REFMAC calculation.",
+                        "context": "dictionary"
+                     }
+                  ]
+               },
+               "I_over_sigma": {
+                  "type": "string",
+                  "description": "Each reflection has an intensity (I) and an uncertainty in measurement \n(sigma(I)), so I/sigma(I) is the signal-to-noise ratio. This\nratio decreases at higher resolution. <I/sigma(I)> is the mean of individual I/sigma(I)\nvalues. Value for outer resolution shell is given in parentheses. In case\nstructure factor amplitudes are deposited, Xtriage estimates the intensities\nfirst and then calculates this metric. When intensities are available in the\ndeposited file, these are converted to amplitudes and then back to intensity\nestimate before calculating the metric.  \nX-ray entry specific, calculated by Phenix Xtriage program.",
+                  "rcsb_description": [
+                     {
+                        "text": "Each reflection has an intensity (I) and an uncertainty in measurement \n(sigma(I)), so I/sigma(I) is the signal-to-noise ratio. This\nratio decreases at higher resolution. <I/sigma(I)> is the mean of individual I/sigma(I)\nvalues. Value for outer resolution shell is given in parentheses. In case\nstructure factor amplitudes are deposited, Xtriage estimates the intensities\nfirst and then calculates this metric. When intensities are available in the\ndeposited file, these are converted to amplitudes and then back to intensity\nestimate before calculating the metric.  \nX-ray entry specific, calculated by Phenix Xtriage program.",
+                        "context": "dictionary"
+                     }
+                  ]
+               },
+               "PDB_R": {
```

Comment:
I meant to comment on this in my previous review, not sure if the comment got lost. There seems to be an overlap between "PDB_" attributes and what we already load.

Reply:
Got it @valasatava. Fixed it in the latest commit.